osu.Game/Beatmaps/Formats/LegacyDecoder.cs

```diff
@@ -78,6 +78,13 @@ private void handleColours(T output, string line)
             if (split.Length != 3)
                 throw new InvalidOperationException($@"Color specified in incorrect format (should be R,G,B): {pair.Value}");
 
+            // Remove characters after the third one
+            for (var i = 0; i<split.Length; i++)
+            {
+                if (split[i].Length > 3)
+                    split[i] = split[i].Remove(3);
+            }
+
```

Comment:
This doesn't work if a part has leading white spaces.
Also, I think, in order to dealing with comments, we should delete comments at somewhere up, not here.

Reply:
I thought it was seperated by newlines for some reason, i agree.